lib/sinon/call.js

```diff
@@ -35,32 +34,31 @@ var callProto = {
     },
 
     calledWith: function calledWith() {
-        var l = arguments.length;
-        if (l > this.args.length) {
+        var self = this;
+        var calledWithArgs = Array.prototype.slice.call(arguments);
```

Comment:
There is a `slice` variable at the top of this file because it's used so frequently. Do you prefer using the lengthy version to be more explicit?